cli/dcoscli/tables.py

```diff
@@ -863,7 +863,7 @@ def slave_table(slaves, field_names=()):
             heading, field_name = field_name.split(':', 1)
         else:
             heading = field_name
-        fields[heading.upper()] = _dotted_itemgetter(field_name.lower())
```

Comment:
Oh yeah, the field name as displayed in the table is still uppercase.

The reason for this is to not assume that fields are all lowercase when we retrieve them from the dicts, because some are not:

```
$ dcos node --json
  {
    "TASK_ERROR": 0,
    "TASK_FAILED": 185,
    "TASK_FINISHED": 14,
    "TASK_KILLED": 6,
    "TASK_KILLING": 0,
    "TASK_LOST": 0,
    "TASK_RUNNING": 6,
    "TASK_STAGING": 0,
    "TASK_STARTING": 0,
    "active": true,
...
```